server/cache.go

```diff
@@ -461,12 +464,92 @@ func (c *clusterInfo) getStoreCount() int {
 	return c.stores.getStoreCount()
 }
 
+func (c *clusterInfo) getStoreWriteStatus() map[uint64]uint64 {
+	res := make(map[uint64]uint64)
+	for _, s := range c.getStores() {
+		res[s.GetId()] = s.status.GetBytesWritten()
+		res[0] += s.status.GetBytesWritten()
```

Comment:
remove this line. If you need total written bytes, please sum all the element in the map.